components/merkledb/tests/map.rs

```diff
@@ -142,8 +143,8 @@ fn compare_map(map: &MapIndex<Rc<Fork>, u8, i32>, ref_map: &HashMap<u8, i32>) ->
 }
 
 fn compare_proof_map(
```

Comment:
Off-topic: The `compare_proof_list_to_vec` test at the bottom of the file is named incorrectly (the correct name should be something like `compare_proof_map_to_hash_map`). Could you fix it?

Reply:
Yes, sure.